public/js/public/details.js

```diff
@@ -44,3 +44,41 @@ function loadPage() {
 }
 
 loadPage();
+function recommendProduct() {
+    $('.list-recomment-product').each(function(){
+        if ( $().owlCarousel ) {
```

Comment:
$().owlCarousel là gì e?

Reply:
cái này e cx lấy của team FE thôi a...chắc là $(this).owlCarousel ạ